__tests__/api.test.js

```diff
@@ -71,6 +71,91 @@ describe('GET /api/reviews', () => {
                 expect(res.body.msg).toBe('Ooops, nothing to see here!')
             })
     });
+
+    test('can filter by category', () => {
+        return request(app)
+        .get(`/api/reviews/?category=dexterity`)
+        .expect(200)
+        .then((reviews) => {
+            reviews.body.forEach(element => {
+                expect(element.category).toBe('dexterity')
+            })
+        })
+    });
+   
+    test('can filter by category with space', () => {
+        return request(app)
+        .get(`/api/reviews/?category='social deduction'`)
+        .expect(200)
+        .then((reviews) => {
+            reviews.body.forEach(element => {
+                expect(element.category).toBe('social deduction')
+            })
+        })
+    });
+    
+    test('can sort asc', () => {
+        return request(app)
+        .get(`/api/reviews/?order=asc`)
+        .expect(200)
+        .then((res) => {
+            let check = true
```

Comment:
It can always be tricky to include more logic in the tests - consider using the jest-sorted package for stuff like this